app/Http/Controllers/PullRequestController.php

```diff
@@ -46,4 +46,20 @@ public function store(StorePullRequestRequest $request): RedirectResponse
 
         return to_route('pull-requests.index');
     }
+
+    /**
+     * Remove the specified resource.
+     *
+     * @param PullRequest $pullRequest
+     *
+     * @return RedirectResponse
+     */
+    public function destroy(PullRequest $pullRequest): RedirectResponse
+    {
+        $this->authorize('delete', $pullRequest);
```

Comment:
its good but at the future, move to $this->authorizeResource() to easly check all method.

Reply:
Couldn't we validate this at an earlier stage like on-request validation?

Reply:
I work on it, i check at the controller __construct with $this->authorizeResource() directly !